docs/dev/developer-guides/aa.md

```diff
@@ -36,13 +30,10 @@ The account abstraction protocol on zkSync is very similar to [EIP4337](https://
 
 ### Keeping nonces unique
 
-::: warning Changes are expected
-
-The current model has some important drawbacks: it does not allow custom wallets to send multiple transactions at the same time, while keeping a deterministic ordering. For
-EOAs the nonces are expected to be sequentially growing, while for the custom accounts the order of transactions can not be determined for sure.
-
-In the future, we plan to switch to a model where the accounts could choose whether they would want to have sequential nonce ordering (the same as EOA) or they want to have arbitrary ordering.
-
+::: warning
```

Comment:
BTW, we have already switched to the model described in the warning. The problem is that we don't yet support custom nonce ordering on the server side : ) I.e. for now the ordering is de facto always sequential

Reply:
So they can choose between sequential or arbitrary ... but always get sequential?

Reply:
@StanislavBreadless correct me if I am wrong - if they send a custom bounce, we just receive the txs in the order they were sent, apply a server bounce to it (which is always a number, and not a arbitrary string) and process them in the order they were received.

Is that it?

Reply:
@githubdoramon @kailash-manasarovar  we rather process them in the same order as accounts with sequential nonces. I.e. not only in the order they were received, but really require the `nonce-1` to be processed before `nonce`. They do not necessarily come to the API in the same order. 

@kailash-manasarovar yeah, so they may select to have custom nonces, but generally it is not much more than a recommendation to the operator, since the operator anyway has discretion to put transactions in any order.

Reply:
ok, thanks, @githubdoramon happy to resolve?